docs/backends/BCS.md

```diff
@@ -214,6 +216,17 @@ default-runtime-attributes {
 ```
 - `docker-image-with-tag` - Docker image stored in Alibaba Cloud Container Registry, such as `registry.cn-shanghai.aliyuncs.com/batchcompute/myubuntu:0.2`.
 
+#### vpc
```

Comment:
Probably it's worth mentioning that defining `vpc` in the configuration is required for the ISV support.

Reply:
`vpc` is  is not related to the isv support, I just supplemented the documentation.